src/main/java/ru/rosatom/documentflow/services/impl/DocTypeServiceImpl.java

```diff
@@ -77,4 +80,15 @@ public DocType attributeToType(Long docTypeId, Long docAttributeId) {
 
         return docTypeRepository.save(docType);
     }
+
+    public boolean isAllowedType(Long id, User user) {
+        boolean alllowed = Objects.equals(getDocTypeById(id).getUserOrganization().getId(), user.getOrganization().getId());
+        return alllowed;
+    }
+
+    public boolean isAllowedTypeAttribute(Long docTypeId, Long docAttributeId, User user) {
+        boolean alllowed = Objects.equals(getDocTypeById(docTypeId).getUserOrganization().getId(), user.getOrganization().getId()) &&
+                Objects.equals(docAttributeRepository.findById(docAttributeId).get().getOrganization().getId(), user.getOrganization().getId());
+        return alllowed;
```

Comment:
название переменной лучше подойдёт isAllowed
```suggestion
        boolean isAlllowed = Objects.equals(getDocTypeById(id).getUserOrganization().getId(), user.getOrganization().getId());
        return isAlllowed;
    }

    public boolean isAllowedTypeAttribute(Long docTypeId, Long docAttributeId, User user) {
        boolean isAlllowed = Objects.equals(getDocTypeById(docTypeId).getUserOrganization().getId(), user.getOrganization().getId()) &&
                Objects.equals(docAttributeRepository.findById(docAttributeId).get().getOrganization().getId(), user.getOrganization().getId());
        return isAlllowed;
```